.mocharc.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   timeout: 25000,
   spec: 'test/**/*.spec.ts',
+  // TODO: This shouldn't be necessary, but the tests on node20 hang, while they pass on node 16 & 18.
+  // `leaked-handles` showed an unclosed docker deamon connection.
```

Comment:
```
no of handles 4

tcp stream {
  fd: 23,
  readable: true,
  writable: true,
  address: {},
  serverAddr: null
}

tcp stream {
  fd: 25,
  readable: true,
  writable: true,
  address: {},
  serverAddr: null
}

tcp stream {
  fd: 28,
  readable: true,
  writable: true,
  address: {},
  serverAddr: null
}

http handle leaked at one of: 
    at h.request (.../balena-compose/node_modules/docker-modem/lib/http.js:51:52)
http stream {
  fd: 30,
  readable: true,
  writable: true,
  address: {},
  method: 'POST',
  path: '/build',
  host: 'localhost'
}
```

Reply:
should we raise an issue with docker-modem x node20 and reference it in the comment here?

Reply:
See: https://github.com/balena-io-modules/balena-compose/pull/38

Reply:
@klutchell I'm not convinced it's an issue w/ docker-modem, b/c once I removed all `deamonStream.unpipe()` things started working again.